src/mpid/ch4/netmod/ucx/ucx_am.h

```diff
@@ -405,28 +405,4 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_am_send_hdr_reply(MPIR_Context_id_t contex
     goto fn_exit;
 }
 
-MPL_STATIC_INLINE_PREFIX int MPIDI_NM_am_recv(MPIR_Request * req)
```

Comment:
👍 to this change. I was planning to do this in #4293, as well. This is part of the generic protocol.